bedrock/products/templates/products/vpn/platforms/desktop.html

```diff
@@ -22,13 +22,13 @@
 {% set _params = '?utm_source=' ~ _utm_source ~ '&utm_medium=referral&utm_campaign=' ~ _utm_campaign %}
 
 {% block page_css %}
-  {{ css_bundle('mozilla-vpn-platforms') }}
+  {{ css_bundle('mozilla-vpn-landing') }}
```

Comment:
This PR will remove the need for the platform-specific CSS bundle. Does it make sense to use `-landing`, or should I re-fork the styles into the `platforms` subfolder?

Reply:
I tried to make things modular so you could compile your own bundle, as opposed to rely on the stylesheet for another page. But if there are things in landing.scss that you need, then we should make those shared components.

Reply:
Copy that. I went back and removed landing from the bundle. It was not needed!